ssl/quic/quic_port.c

```diff
@@ -101,16 +101,20 @@ CRYPTO_MUTEX *ossl_quic_port_get0_mutex(QUIC_PORT *port)
     return port->mutex;
 }
 
-static OSSL_TIME get_time(void *arg)
+OSSL_TIME ossl_quic_port_get_time(QUIC_PORT *port)
 {
-    QUIC_PORT *port = arg;
-
     if (port->now_cb == NULL)
         return ossl_time_now();
 
     return port->now_cb(port->now_cb_arg);
 }
 
+static OSSL_TIME get_time(void *port)
+{
+    return ossl_quic_port_get_time(port);
```

Comment:
Why do we need this wrapper, rather than just calling `ossl_quic_port_get_time()` directly where it is used?

Reply:
It is a different type. `ossl_quic_port_get_time` has an argument of type `QUIC_PORT *` not `void *`. So technically calling one from a function pointer with the other type is UB. (See also @nhorman's recent work on fixing related issues in LHASH)

I will add an explicit cast for exposition.